WebRoot/css/zm.css

```diff
@@ -3310,38 +3296,40 @@ HTML>BODY .appt-selected .appt_allday_body
 	text-align:center;
 }
 
+.ZmSchedulerGridHeaderLeft {
+	@LeftSeparator@
+	@AlignLeft@
+	padding-left:@SkinViewGutterSpace@;
+}
+
+.ZmSchedulerGridHeaderLeft > div{
+	position:absolute;
```

Comment:
Don't we need other supporting property here? ie `top`, `left`.. etc?

Reply:
Not required, div is aligned based on its "relative" parent. Not specifically required a value here.